blogposts/2021/release-3.34

```diff
@@ -0,0 +1,84 @@
+---
+title: "Sourcegraph 3.34 release"
+publishDate: 2021-11-23T10:00-07:00
+description: "Sourcegraph 3.34 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.34"
+published: false
+heroImage:
+socialImage:
+changelogItems:
+
+---
+
+Sourcegraph 3.34 is now available! For this release, we introduced:
+
+## No results page
+When your search has no results, we will now display helpful tips on how to use Sourcegraph search, examples of how to solve common query issuses, and links to help videos addressing common search use cases.
+
+
```

Comment:
```suggestion
## Find Implementations for Go
Navigating to implementations is now available for Go! If you have precise code intelligence enabled in your Go repositories, you can now click on Find Implementations to navigate to a symbol's interface definition. If you're at the interface definition itself, clicking on Find Implementations will show all the places where the interface is being implemented, allowing you to explore how it's being used by other users across repositories. It will also show you which interfaces a struct implements. 
<img width="1193" alt="Screenshot 2021-11-22 at 18 28 39" src="https://user-images.githubusercontent.com/1657213/142938393-7aed0c41-28b8-4cab-bf0d-2f9fd7f1078c.png">
<img width="1193" alt="Screenshot 2021-11-22 at 18 29 03" src="https://user-images.githubusercontent.com/1657213/142938404-3b845bec-5da2-4ec7-bf17-0e273ff38dcd.png">
```